Legacy table for anatomical MRI data (#58)
bids-standard/bep001: ProposedChangesToSpecification.md

* ManufacturerModelName
* InstitutionName 
* PulseSequenceType
* PulseSequenceDetails
* EstimationPaper
* EstimationAlgorithm
* EstimationSoftwareName
* EstimationSoftwareVersion 
* EstimationSoftwareLanguage
* EstimationSoftwareEnv 

## Sidecar JSON files of grouping suffixes 

Mention about root JSON + repeating JSON file convention.
Which root JSON metadata fields are optional/required. 

## Legacy suffixes for anatomical data
### Proposed Change 
Moves some suffixes for `anatomical data` to a new table called "LEGACY suffixes". These suffixes are still valid BIDS format,
but we reccomend not to use them anymore. They are ambiguous and/or inconsistent.

### Justification 
The following suffixes are moved to the legacy table:
 * **`_T2star`**: what is meant is T2\*-_weighted_ data. To be consistent with the suffixes `T1w`, `T2w` and `PDw`, we want
to use the suffix `_T2starw` for this.
 * **`_FLAIR`**: This is a sequence name rather than a contrast or a way of grouping quantitative maps. We prefer to use
`_T2w`, `_PDw`, or `_MESE` depending on the use case.
 * **`_FLASH`**: This is a sequence name rather than a contrast or a way of grouping quantitative maps. We prefer to use

> `_T2starw`, `_PDw`, or `_GREME` depending on the use case.

Reviewer: ```suggestion
`_T2starw`, `_PDw`, or `_MEGRE` depending on the use case.
```

Reviewer 2: It would be really good to get some highly-clinically oriented people to comment on this.  For white matter lesions (whether vanilla white matter hyperintensities or multiple sclerosis lesions) particular, FLAIR is so ubiquitous that it’s important to spell out exactly what is recommended (and that there is buy-in from the community that uses these scans).

In particular, I have never heard of a PD-weighted FLAIR image.  A concrete replacement for `_FLAIR` as typically encountered should be offered.  I think this is something like:

```
For a typical FLAIR with T2-weighting but suppression of the CSF, 
we recommend `acq-FLAIR_T2w`.
```

Conversely, I’ve never heard of FLASH being used for T2 weighting, no less T2star... AFAIK, in its usual instantiation, it’s used for T1-weighted imaging.  However, here, unlike FLAIR, FLASH just speeds things up and doesn’t change the contrast, so maybe it’s not so important to convey this. (Not 100% confident on this).

**Can I perhaps make a radical suggestion**:  As I understand it, the proposal to remove FLAIR and FLASH are not specific to the remit of BEP001, right?  They are rather part of a general revision/cleaning of the BIDS standard.  *UNLESS* these changes are key to the multi-echo use case, can I propose that you create a *separate* PR, distinct from BEP001, that can get a fuller airing in the BIDS community, on the future of sequenced-named modality labels?  BIDS was not crafted 20 years ago, but just a few years ago, and a wide community of people thought that, despite the slight inconsistency, there was merit in using these particular labels.

@KirstieJane, @Gilles86, @Remi-Gau, @franklin-feingold... what do you think?

Reviewer 3: I am concerned (not opposed) to dropping support of FLAIR and FLASH support in modalities supported. I find these sequences are prevalent (as you mentioned especially in the clinical world). I feel raising a larger community discussion may be best to hear more opinions on this. I do not believe this could uptaken until BIDS 2.0 (due to backward compatibility) anyways, but I think great to hear from more of the community on this.

Reviewer: @Gilles86, @agahkarakuzu and I are just chatting about this now - sorry to have dropped the thread for so long.

@nicholst - you've made a strong case for keeping FLAIR in the list of common scan acquisitions. We might tag you in the future for help adding a description to make it clear that we recommend using `FLAIR` only when the scan is T2-weighted with suppression of the CSF.

We'd like to still move FLASH to the legacy table because there are so many different flavours of this acquisition that it is impossible to confidently combine them across datasets. ie if data set 1 has a FLASH scan, there's little chance that it will look the same - have the same contrast - as the FLASH scan in data set 2.

So proposal: keep FLAIR in a prominant position, move FLASH, correct the labels that should have had `w`s after them going forwards (and put the non-`w`-d labels in the legacy table with the same meaning).

 * **`_PD`**: what is meant is PD*-_weighted_ data. To be consistent with the suffixes `T1w`, `T2w` and `T2starw`, we want
to use the suffix `_PDw` for this.

>  * **`_PDT2`**: We recommend to use `PDT2map` for this.

Reviewer: @nicholst - we're not super clear on what the `PDT2` scan _is_ and so we're not super sure about how to classify it! Can you help us? (References fine too!)

Reviewer 2: PD T2 “dual echo” is just something that I’ve always seen with clinical scans.  I found this reference: [Fast Spin Echo Dual Echo](http://mriquestions.com/dual-echo-fse.html), writing

> For the first 20 years of clinical MRI, double-echo sequences were part of standard protocols for nearly all body parts. Short and long echo times (TE) were used to generate proton-density and T2-weighted images in a single acquisition using a long repetition time (TR). Over the last decade T2-FLAIR has largely replaced proton-density sequences for neuroimaging, but they are still used in musculoskeletal and other body applications.

While it may seem like an anachronism this dual echo pair is very common in existing clinical datasets, especially for clinical trials where the protocol was specified long ago.

Reviewer: Hi @nicholst!! Thank you so much for commenting here - I'm so sorry that we've dropped this thread for so long.

@agahkarakuzu, @Gilles86 and I are just reviewing this pull request and wondering about whether this scan is _one_ file (maybe with two volumes in it - in the 4th dimension) or two files? Or are the two echos combined in someway to one 3D file?

Thank you!

Reviewer 2: > @agahkarakuzu, @Gilles86 and I are just reviewing this pull request and wondering about whether this scan is one file (maybe with two volumes in it - in the 4th dimension) or two files? Or are the two echos combined in someway to one 3D file?

IMHO, the `PDT2` suffix only makes sense for a 2-volume image... if the images are separated out, then one would be `_PD` and the other `_T2`.  _However_, `heudiconv` presently uses it with 3D images too:  `dcm2nii` creates 2 3D images, but `heudiconv` uses a logic that requires images sharing a DICOM Series UID to share a BIDS base and image type;  so since both the PD and T2 arise from the same series, they get the same base and then must be distinguished with (e.g.) `echo-1` and `echo-2`.

Does this help?

 * **`_inplaneT1`**: this is not a proper MRI contrast or grouping variable. The fact that the data is acquired with the

> same prescription as the functional data is already decsribed in the orientation matrix. Furthermore,

Reviewer: ```suggestion
same prescription as the functional data is already decsribed in the orientation matrix.
```

> it could be communicated via the `acq-`-tag.

Reviewer: ```suggestion
We recommend that this information is communicated via the `acq-`-tag.
```

 * **`_inplaneT2`**: this is not a proper MRI contrast or grouping variable. The fact that the data is acquired with the

> same prescription as the functional data is already decsribed in the orientation matrix. Furthermore,

Reviewer: ```suggestion
same prescription as the functional data is already decsribed in the orientation matrix.
```

> it could be communicated via the `acq-`-tag.

Reviewer: ```suggestion
We recommend that this information is communicated via the `acq-`-tag.
```
